managed-databases/postgresql-and-mysql/concepts.mdx

```diff
@@ -31,21 +31,39 @@ Database Instances have settings that allow you to tune the behavior of their da
 
 <Concept opened>
 
+## Basic Block Storage
+
+Basic Block Storage is a volume type. With this type your storage is decoupled from your compute resources. You can increase your storage space without changing your node type. You can define your storage capacity by entering the desired amount in the box.
+
+</Concept>
+
+<Concept opened>
+
+## Block Storage Low Latency
+
+Low Latency is a storage type that works like the [Basic Block Storage](#basic-block-storage), while providing enhanced efficiency. You can choose between **5k** or **15k IOPS** according to your workloads input/output operations needs, with lower latency and high resiliency. You can increase your volume to up to 10 TB.
```

Comment:
```suggestion
Block Storage Low Latency is a storage type that works like the [Basic Block Storage](#basic-block-storage), while providing enhanced efficiency. You can choose between **5k** or **15k IOPS** according to your workloads input/output operations needs, with lower latency and high resiliency. You can increase your volume to up to 10 TB.
```

Reply:
15k is not yet available. Please wait a bit so we can indicate it or not for the beta